.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -40,6 +40,8 @@ body:
           required: true
         - label: This isn't a runtime feature (e.g. library functionality, non-ECMAScript syntax with JavaScript output, new syntax sugar for JS, etc.)
           required: true
+        - label: This isn't a request to add a new utility type
```

Comment:
I think we need a FAQ entry to link to. Thoughts?

Reply:
https://github.com/microsoft/TypeScript/issues/39522#issuecomment-656201388 or https://github.com/microsoft/TypeScript/issues/55301? They aren't FAQs per say. Otherwise, the wiki somewhere?

Reply:
Wrote https://github.com/microsoft/TypeScript/wiki/No-New-Utility-Types

Reply:
```suggestion
        - label: This isn't a request to add a new utility type: https://github.com/microsoft/TypeScript/wiki/No-New-Utility-Types
```

Reply:
Added to the template, PTAL